spec/unit/lib/temporal/grpc_client_spec.rb

```diff
@@ -28,12 +28,61 @@
           execution_timeout: 0,
           run_timeout: 0,
           task_timeout: 0,
-          memo: {}
+          memo: {},
+          workflow_id_reuse_policy: :allow,
         )
       end.to raise_error(Temporal::WorkflowExecutionAlreadyStartedFailure) do |e|
         expect(e.run_id).to eql('baaf1d86-4459-4ecd-a288-47aeae55245d')
       end
     end
+
+    it 'starts a workflow with scalar arguments' do
```

Comment:
There was a test like this one for signal_with_start, but there wasn't one for start. I wanted to add workflow_id_reuse_policy to the args/asserts for both call paths.